x-pack/legacy/plugins/monitoring/public/components/elasticsearch/shard_allocation/_shard_allocation.scss

```
@@ -1,65 +1,52 @@
-// SASSTODO: Generic selector
-monitoring-shard-allocation {
-  display: block;
-  border-top: $euiSizeS solid $euiColorLightestShade;
-}
-
 .monClusterTitle {
   font-size: $euiFontSizeL;
   margin: 0;
 }
 
 // SASSTODO: This needs a full rewrite / redesign
 .monCluster {
-  cluster-view {
-    display: block;
-  }
-  .parent {
-    padding-top: 14px;
-    border-left: 3px solid $euiColorSuccess !important;
-    &.red {
-      border-left: 3px solid $euiColorDanger !important;
-    }
-    &.yellow {
-      border-left: 3px solid $euiColorWarning !important;
-    }
-  }
-  td.unassigned {
+  .monUnassigned {
     vertical-align: middle;
     width: 150px;
   }
 
-  .child {
+  .monUnassigned__children,
+  .monAssigned__children {
+    padding-top: $euiSizeL;
+  }
+
+  .monChild {
     float: left;
     align-self: center;
+    background-color: $euiColorLightestShade;
+    margin: $euiSizeS;
+    border: 1px solid $euiColorMediumShade;
+    border-radius: $euiSizeXS;
+    padding: $euiSizeXS/2 0;
 
     &.index {
       border-left: $euiSizeXS solid $euiColorSuccess;
-      &.red {
+
+      &.monChild--danger {
         border-left: $euiSizeXS solid $euiColorDanger;
       }
-      &.yellow {
+
+      &.monChild--warning {
         border-left: $euiSizeXS solid $euiColorWarning;
       }
     }
-    background-color: $euiColorDarkShade;
-    margin: 5px;
-    .title {
-      padding: 5px 7px;
-      display: inline-block;
+
+    .monChild__title {
+      padding: $euiSizeXS $euiSizeS;
       text-align: center;
-      font-size: 12px;
-      font: 10px sans-serif;
+      font-size: $euiSizeM;
```

We actually have font-size specific variables instead of using the base sizing ones. The equivalent here would be:
```suggestion
      font-size: $euiFontSizeXS;
```